### PR TITLE
Add fuzzy matching support to the vimgrep command

### PR DESCRIPTION
**Pull request by yegappan**

### CODE REVIEW

**Comment by vim-ml** [comment]
Hi,

The Github actions for this PR didn't run because it says
"This workflow is awaiting approval from a maintainer".
I haven't seen this message before.

Regards,
Yegappan

**Comment by chrisbra** [comment]
not sure why this is needed now. I approved the GH workflow

File: src/quickfix.c [hunk 1]
```diff
@@ -5920,45 +5921,91 @@ vgr_match_buflines(
...
+			    duplicate_name ? 0 : buf->b_fnum,
+			    ml_get_buf(buf,
+				regmatch->startpos[0].lnum + lnum, FALSE),
+			    regmatch->startpos[0].lnum + lnum,
+			    regmatch->startpos[0].col + 1,
+			    FALSE,      // vis_col
+			    NULL,	    // search pattern
+			    0,	    // nr
+			    0,	    // type
+			    TRUE	    // valid
```
Comment (dpelle) [inline src/quickfix.c]: Comments look misaligned.

**Comment by vim-ml** [comment]
Hi Christian,

On Sun, Apr 25, 2021 at 11:21 PM Christian Brabandt <
***@***.***> wrote:

> not sure why this is needed now. I approved the GH workflow
>
>
> It looks like Github changed the policies:

https://github.blog/changelog/2021-04-22-github-actions-maintainers-must-approve-first-time-contributer-workflow-runs/
https://github.blog/2021-04-22-github-actions-update-helping-maintainers-combat-bad-actors/

When I updated the PR, the Github actions now requires maintainer
approval to run again.

- Yegappan

**Comment by Shane-XB-Qian** [comment]
> not sure why this is needed now

some hacker using pr action to dig coin, i guess that's the why, and so check pr (specially for ci 'action') carefully as well.

**Comment by Shane-XB-Qian** [comment]
seems just added `[f]` into quickfix.txt, but did not mentioned what's that use for..
// supposed would update at next runtime update?

**Comment by Shane-XB-Qian** [comment]
seems just added `[f]` into quickfix.txt, but did not mentioned what's that use for..
// supposed would update at next runtime update?

**Comment by chrisbra** [comment]
> https://github.blog/changelog/2021-04-22-github-actions-maintainers-must-approve-first-time-contributer-workflow-runs/
> https://github.blog/2021-04-22-github-actions-update-helping-maintainers-combat-bad-actors/
> 
> When I updated the PR, the Github actions now requires maintainer
> approval to run again.

Thanks, this is unfortunately a bit annoying. I will check the PR page then more actively and see if I need to approve anything.

**Comment by Shane-XB-Qian** [comment]
github network for page and feed recently is abnormal too (or just in some country).
// above 2 links cannot be opened, but I Guess pr action was limited due to that..

**Comment by vim-ml** [comment]
Hi,

On Mon, Apr 26, 2021 at 11:01 PM Shane-XB-Qian ***@***.***>
wrote:

> seems just added [f] into quickfix.txt, but did not mentioned what's that
> use for..
> // supposed would update at next runtime update?
>
>
>
My pull request includes the updates to the help files. I guess it will
show up when Bram pushes the next runtime file updates.

- Yegappan

**Pull request closed without merging**